### ISSUE TITLE
Compatibility issues on macbook pro with M1 chip

### ISSUE THREAD

**Issue by mdoolin2014** [biocore/mmvec#175]
Hello, 

I'm very interested in using your program on my dataset, but I'm having issues getting started. I've tried both the non-qiime2 and qiime2 methods of using the program, but am having compatibility issues both ways. I'm wondering if anyone has experience these issues and has figured out a way to work around them. I've follow instructions online for workarounds for these general errors, specified below. However, much of it involves updating python or tensorflow to newer versions to be compatible with the packages I'm trying to run. Of course, this does not make sense in this scenario since I'm running the version specified within mmvec.  

By making the mmvec_env environment and installing with conda-forge, installation looks successful but then I get the error "zsh: illegal hardware instruction  mmvec --help" when I try to test the functionality. Other forums say that this is likely an issue between tensorflow and python compatibility, but I'm running python v. 3.7.12 as specified in the mmvec environment. 

In qiime2, I installed qiime2-2020.6 in a Rosetta terminal and everything seemed to work until trying to install the mmvec plugin. Once I reset the cache, it returns the error "QIIME is caching your current deployment for improved performance. This may take a few moments and should only happen once per deployment" anytime I try to run a qiime command instead of running the command. 

Any ideas from developers or the community?

**Comment by mortonjt** [comment]
Hi, @mdoolin2014 I'm assuming that you tried reading the installation instructions in the README (don't upgrade python / tensorflow).

I'm wondering if your problem has to do with running this on a Macbook rather than Linux.  I haven't installed MMvec on a Macbook since 2019, since it is fairly computationally intensive.

**Comment by mdoolin2014** [comment]
Yes, definitely following the README instructions and not upgrading. Interesting, ok maybe I'll try running on our cluster instead to see if that'll just work more easily. I'll follow up if I figure it out.

**Comment by mdoolin2014** [comment]
@mortonjt Update that I ran it through qiime2-2020.6 on a linux machine and had no issues. Definitely seems to be an issue with the Mac configuration. Thanks for your suggestion!

**Comment by MOUTAI-Lee** [comment]
Hello , I also encountered compatibility issues with the version. I would like to ask where you downloaded QIIME2_2020. For the tensoflow you mentioned, according to my understanding, you need to find the tensoflow1.14 version from CUND. If you can see my message, I very much hope you can reply to me, thank you!
